apps/opik-backend/src/main/resources/liquibase/db-app-analytics/migrations/000015_change_tables_to_replicated.sql

```diff
@@ -0,0 +1,225 @@
+--liquibase formatted sql
+
+--changeset liyaka:change-tables-to-replicated-01 id:create-automation-rule-evaluator-logs
+CREATE TABLE IF NOT EXISTS ${ANALYTICS_DB_DATABASE_NAME}.automation_rule_evaluator_logs1
+(
+    `timestamp` DateTime64(9, 'UTC') DEFAULT now64(9),
+    `workspace_id` String,
+    `rule_id` FixedString(36),
+    `level` Enum8('TRACE' = 0, 'DEBUG' = 1, 'INFO' = 2, 'WARM' = 3, 'ERROR' = 4),
+    `message` String,
+    `markers` Map(String, String)
+)
+ENGINE = ReplicatedMergeTree('/clickhouse/tables/{shard}/opik_dev/automation_rule_evaluator_logs', '{replica}')
+ORDER BY (workspace_id, rule_id, timestamp)
+TTL toDateTime(timestamp + toIntervalMonth(6))
+SETTINGS index_granularity = 8192;
+--rollback DROP TABLE ${ANALYTICS_DB_DATABASE_NAME}.automation_rule_evaluator_logs1;
+
+--changeset liyaka:change-tables-to-replicated-02 id:migrate-automation-rule-evaluator-logs
+ALTER TABLE ${ANALYTICS_DB_DATABASE_NAME}.automation_rule_evaluator_logs1 ATTACH PARTITION tuple() FROM ${ANALYTICS_DB_DATABASE_NAME}.automation_rule_evaluator_logs;
+DROP TABLE ${ANALYTICS_DB_DATABASE_NAME}.automation_rule_evaluator_logs SYNC SETTINGS max_table_size_to_drop = 0;
+RENAME TABLE ${ANALYTICS_DB_DATABASE_NAME}.automation_rule_evaluator_logs1 TO ${ANALYTICS_DB_DATABASE_NAME}.automation_rule_evaluator_logs;
+--rollback RENAME TABLE ${ANALYTICS_DB_DATABASE_NAME}.automation_rule_evaluator_logs TO ${ANALYTICS_DB_DATABASE_NAME}.automation_rule_evaluator_logs1;
+
+--changeset liyaka:change-tables-to-replicated-03 id:create-comments
+CREATE TABLE IF NOT EXISTS ${ANALYTICS_DB_DATABASE_NAME}.comments1
+(
+    `id` FixedString(36),
+    `entity_id` FixedString(36),
+    `entity_type` Enum8('trace' = 1, 'span' = 2),
+    `project_id` FixedString(36),
+    `workspace_id` String,
+    `text` String,
+    `created_at` DateTime64(9, 'UTC') DEFAULT now64(9),
+    `last_updated_at` DateTime64(9, 'UTC') DEFAULT now64(9),
+    `created_by` String DEFAULT '',
+    `last_updated_by` String DEFAULT ''
+)
+ENGINE = ReplicatedReplacingMergeTree('/clickhouse/tables/{shard}/opik_dev/comments','{replica}',last_updated_at)
```

Comment:
@liyaka we shoudn't hardcode the schema

```suggestion
ENGINE = ReplicatedReplacingMergeTree('/clickhouse/tables/{shard}/${ANALYTICS_DB_DATABASE_NAME}/comments','{replica}',last_updated_at)
```